docs/Middleware-Guide.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+In designing EigenLayer, we aspired to make minimal assumptions about the structure of middlewares built on top of it. If you are getting started looking at the EigenLayer codebase, the Slasher contract contains most of the logic that actually mediates the interactions between EigenLayer and middlewares. Additionally, there is a general-purpose /middleware/ folder, which contains code that can be extended, used directly, or consulted as a reference in building middleware on top of EigenLayer.
+
+## Important Terminology
+### Tasks
+A task in EigenLayer is the discretized unit of work that operators commit to doing when serving a middleware.  
+### Strategies
+A strategy in EigenLayer, its most simple form, is a contract that stakers can use to deposit and delegate their assets to middlewares on EigenLayer. 
```

Comment:
typo: [in] it's most simple form.

not loving this description, especially since the "delegate" part isn't managed by strategies.

perhaps
"A **strategy** in EigenLayer is a contract that holds staker deposits, i.e. it controls one or more restaked asset(s). At launch EigenLayer will feature only simple strategies which idly hold a single token. However, EigenLayer's strategy design is flexible, and in the future strategies could be deployed which implement more complex logic, including DeFi integrations."